a1: 0.5
b1: 0.5
c1: 0.5
zero_v1: 3.0
a2: 0.5
b2: 0.5
c2: 0.5
zero_v2: 4.0
a3: 0.5
b3: 0.5
c3: 0.5
zero_v3: 3.0
a4: 0.5
b4: 0.5
c4: 0.5
zero_v4: 6.0
a5: 0.5
b5: 0.5
c5: 0.5
zero_v5: 4.0
a6: 0.5
b6: 0.5
c6: 0.5
zero_v6: 6.0
a7: 0.5
b7: 0.5
c7: 0.5
zero_v7: 9.0
a8: 0.5
b8: 0.5
c8: 0.5
zero_v8: 8.0
a9: 0.5
b9: 0.5
c9: 0.5
zero_v9: 7.0
